tensorflow/python/kernel_tests/conv_ops_test.py

```diff
@@ -319,6 +319,20 @@ def testConv2D2x2FilterStride1x2(self):
                        strides=[1, 2], padding="VALID",
                        expected=expected_output)
 
+  def testConv2D1x1FilterStride2(self):
+    expected_output = [1.0, 3.0, 7.0, 9.0]
+    self._VerifyValues(tensor_in_sizes=[1, 3, 3, 1],
+                       filter_in_sizes=[1, 1, 1, 1],
+                       strides=[2, 2], padding="VALID",
+                       expected=expected_output)
+
+  def testConv2D1x1FilterStride2Same(self):
```

Comment:
This test isn't exactly testing anything different than SAME -- perhaps modify the example to exercise this?  Maybe try:

tensor_in_sizes=[1, 5, 5, 1]
filter_in_sizes=[1, 2, 2, 1]
strides=[2, 2]

So that the expected result is that you have some padding to exercise the difference between same and valid padding.